extensions/wikia/DesignSystem/services/templates/DesignSystemGlobalNavigationOnSiteNotifications_index.php

```diff
@@ -0,0 +1,19 @@
+<div id="onSiteNotificationsEntryPoint"
+     class="wds-global-navigation__notifications-menu wds-global-navigation__notifications-menu wds-dropdown notifications-container">
+	<div class="wds-dropdown__toggle wds-global-navigation__dropdown-toggle"
+	     title="<?= wfMessage( 'global-navigation-notifications-title' )->escaped() ?>">
+		<div class="bubbles">
+			<div class="wds-global-navigation__notifications-menu-counter on-site-notifications-count">
```

Comment:
If `on-site-notifications-count` is specifically for referencing this element in JS and not for styling then I would move it to the `id` attribute.